src/main/java/org/opencds/cqf/providers/CqlExecutionProvider.java

```diff
@@ -241,7 +241,7 @@ public Bundle evaluate(
         context.registerLibraryLoader(getLibraryLoader());
 
         if (parameters != null){
-            for (ParametersParameterComponent pc : parameters.getParameter())
+            for (org.hl7.fhir.dstu3.model.Parameters.ParametersParameterComponent pc : parameters.getParameter())
```

Comment:
Why was this change required, and does it impact the version-portability of the CqlExecutionProvider?

Reply:
The cqf-ruler doesn't compile without this change on my machine. This fixes an error in the original PR for the CQL execution provider. The original PR did not include a version increment either, so the version impact is the current unreleased revision of cqf-ruler on the master branch is broken, while the revision on this branch compiles.

Reply:
Huh... It is weird that it is compiling on my machine, but an error shows up when I open the class.